lambdas/service/app.py

```diff
@@ -1475,6 +1475,12 @@ def get_order():
                     'schema': {'type': 'string'},
                 },
             },
+        },
+        '410': {
+            'description': format_description('''
+                The manifest associated with the `objectKey` in this request has
+                expired, i.e. `GoneError`. Request a new manifest.
```

Comment:
`GoneError` seems like a Python exception because of the formatting, but I don't think it is one. The 410 response does mean "gone" which I think is what you're getting at, but I think that's clear enough from the rest of the message and the status code. I suggest just removing "i.e. `GoneError`".